plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java

```diff
@@ -51,7 +43,7 @@ public String getName()
     @Override
     public void close()
     {
-        sqlExecutor.execute("DROP TABLE " + table.getName());
+        table.close();
```

Comment:
pre-existing: shouldn't we drop the view first and then the underlying relation?

Reply:
will fix too